content/docs/hooks-overview.md

```diff
@@ -153,32 +153,32 @@ function FriendStatusWithCounter(props) {
+Hooks आपको एक कौम्पोनॅन्ट में साइड इफेक्ट को व्यवस्थित करने देते हैं जो pieces से संबंधित होते हैं (जैसे कि सब्सक्रिप्शन लेना और हटाना), बजाय lifecycle के तरीकों के आधार पर विभाजन को मजबूर करने के।
 
->Detailed Explanation
+>विस्तृत विवरण
 >
->You can learn more about `useEffect` on a dedicated page: [Using the Effect Hook](/docs/hooks-effect.html).
+>आप एक समर्पित पेज पर `useEffect` के बारे में अधिक जान सकते हैं: [Effect Hook का उपयोग करना](/docs/hooks-effect.html)
 
-## ✌️ Rules of Hooks {#rules-of-hooks}
+## ✌️ Hook के नियम {#rules-of-hooks}
 
-Hooks are JavaScript functions, but they impose two additional rules:
+Hooks JavaScript फंक्शन हैं, लेकिन वे दो अतिरिक्त नियम लागू करते हैं:
 
-* Only call Hooks **at the top level**. Don’t call Hooks inside loops, conditions, or nested functions.
-* Only call Hooks **from React function components**. Don’t call Hooks from regular JavaScript functions. (There is just one other valid place to call Hooks -- your own custom Hooks. We'll learn about them in a moment.)
+* Hooks **सबसे उप्पर** कॉल करें। लूप्स, कंडीशंस, या नेस्टेड फ़ंक्शन के अंदर Hook को कॉल न करें।
+* Hooks सिर्फ ** React फ़ंक्शन कौम्पोनॅन्ट ** मैं ही कॉल करें। नियमित जावास्क्रिप्ट फ़ंक्शंस से Hook कॉल न करें। (Hooks को कॉल करने के लिए सिर्फ एक अन्य वैध स्थान है -- अपने स्वयं के कस्टम Hooks। हम उनके बारे में एक क्षण में जानेंगे।)
 
-We provide a [linter plugin](https://www.npmjs.com/package/eslint-plugin-react-hooks) to enforce these rules automatically. We understand these rules might seem limiting or confusing at first, but they are essential to making Hooks work well.
+हम एक [लिंटर प्लगइन](https://www.npmjs.com/package/eslint-plugin-react-hooks) प्रदान करते हैं जो इन नियमों को स्वचालित रूप से लागू कर देते है। हम समझते हैं कि ये नियम पहले सीमित या भ्रमित हो सकते हैं, लेकिन वे Hooks को अच्छी तरह से काम करने के लिए आवश्यक हैं।
 
->Detailed Explanation
+>विस्तृत विवरण
 >
->You can learn more about these rules on a dedicated page: [Rules of Hooks](/docs/hooks-rules.html).
+>आप एक समर्पित पेज पर ये नियम के बारे में अधिक जान सकते हैं: [Hooks के नियम](/docs/hooks-rules.html)
 
-## 💡 Building Your Own Hooks {#building-your-own-hooks}
+## 💡 अपनी खुद की Hooks का निर्माण {#building-your-own-hooks}
 
-Sometimes, we want to reuse some stateful logic between components. Traditionally, there were two popular solutions to this problem: [higher-order components](/docs/higher-order-components.html) and [render props](/docs/render-props.html). Custom Hooks let you do this, but without adding more components to your tree.
+कभी-कभी, हम कौम्पोनॅन्ट के बीच कुछ स्टेटफुल लॉजिक का पुन: उपयोग करना चाहते हैं। परंपरागत रूप से, इस समस्या के दो लोकप्रिय समाधान थे: [हाई आर्डर कौम्पोनॅन्ट](/docs/higher-order-components.html) और [रेंडर प्रॉप्स](/docs/render-props.html)। कस्टम Hooks आपको ऐसा करने देते हैं, लेकिन आपके tree में अधिक कौम्पोनॅन्ट को जोड़े बिना।
```

Comment:
`कस्टम Hooks आपको ऐसा करने देते हैं, लेकिन आपके tree में अधिक कौम्पोनॅन्ट को जोड़े बिना।` => `लेकिन कस्टम Hooks आपको आपके tree में अधिक कौम्पोनॅन्ट को जोड़े बिना ऐसा करने देते हैं।`

Reply:
`जोड़े` => `ऐड करे`

i missed this in previous review. sorry about that